inference-engine/src/hetero_plugin/hetero_plugin.cpp

```diff
@@ -154,23 +155,54 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork &network, const Configs
     return qr;
 }
 
-Parameter Engine::GetMetric(const std::string& name, const std::map<std::string, Parameter> & /*options*/) const {
+Parameter Engine::GetMetric(const std::string& name, const std::map<std::string, Parameter>& options) const {
     if (METRIC_KEY(SUPPORTED_METRICS) == name) {
         IE_SET_METRIC_RETURN(SUPPORTED_METRICS, std::vector<std::string>{
             METRIC_KEY(SUPPORTED_METRICS),
             METRIC_KEY(FULL_DEVICE_NAME),
-            METRIC_KEY(SUPPORTED_CONFIG_KEYS)});
+            METRIC_KEY(SUPPORTED_CONFIG_KEYS),
+            METRIC_KEY(DEVICE_ARCHITECTURE),
+            METRIC_KEY(IMPORT_EXPORT_SUPPORT)});
     } else if (METRIC_KEY(SUPPORTED_CONFIG_KEYS) == name) {
         IE_SET_METRIC_RETURN(SUPPORTED_CONFIG_KEYS, std::vector<std::string>{
             HETERO_CONFIG_KEY(DUMP_GRAPH_DOT),
             "TARGET_FALLBACK",
             CONFIG_KEY(EXCLUSIVE_ASYNC_REQUESTS)});
     } else if (METRIC_KEY(FULL_DEVICE_NAME) == name) {
         IE_SET_METRIC_RETURN(FULL_DEVICE_NAME, std::string{"HETERO"});
+    } else if (METRIC_KEY(IMPORT_EXPORT_SUPPORT) == name) {
+        IE_SET_METRIC_RETURN(IMPORT_EXPORT_SUPPORT, true);
+    } else if (METRIC_KEY(DEVICE_ARCHITECTURE) == name) {
+        auto deviceIt = options.find("TARGET_FALLBACK");
```

Comment:
these `options` come from `LoadNetwork` in `ie_core.cpp`. But what if we use the following code:
```
ie::Core core;
core.SetConfig({ { "TARGET_FALLBACK", "CPU" } }, "HETERO");
... 
```
In this case we have value for `TARGET_FALLBACK`, but `options` are still empty.

Reply:
Fixed